corgi/core/models.py

```diff
@@ -443,6 +443,13 @@ class ProductStream(ProductModel, TimeStampedModel):
 
     cpe = models.CharField(max_length=1000, default="")
 
+    # NOTE brew_tags and yum_repositories values shouldn't be exposed outside of Red Hat
+    brew_tags = models.JSONField(default=dict)
```

Comment:
It is fine and good to add specific attributes to ProductStream ... mentally I read the ProductModel base
as RedHatProductModel.